oop/src/views/ReviewPage.vue

```diff
@@ -82,6 +84,15 @@ export default {
 
           switch (returnCode) {
             case '0': // Submission successful.
+              // Append city and state into one field - addressLines[1] if country is USA
+              if(this.$store.state.form.country == 'United States'){
+                const appendedCityState = this.$store.state.form.addressLines[1].value + ' ' + this.$store.state.form.province + ' USA';
+                formState.addressLines[1].value = appendedCityState;
+                formState.province = null;
+                this.$store.dispatch(formModule + '/' + SET_ADDRESS_LINES, formState.addressLines);
+                this.$store.dispatch(formModule + '/' + SET_PROVINCE, formState.province);
```

Comment:
I think you're still modifying the state. Can you change this to the following?
```
const addressLines = [...formState.addressLines];
addressLines[1].value = appendedCityState;
this.$store.dispatch(formModule + '/' + SET_ADDRESS_LINES, addressLines);
this.$store.dispatch(formModule + '/' + SET_PROVINCE, null);
```
I think this should work, but please confirm.